internal/cli/internal/command/services/postgres/util.go

```diff
@@ -0,0 +1,51 @@
+package postgres
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/azazeal/pause"
+	"github.com/superfly/flyctl/internal/client"
+	"github.com/superfly/flyctl/internal/logger"
+	"github.com/superfly/flyctl/pkg/iostreams"
+)
+
+func waitForMachineState(parent context.Context, client *client.Client, appID, machineID, state string) error {
+
```

Comment:
I've noticed a lot of empty lines between function declaration and first line of body - this isn't something one sees in Go codebases.